site/kubernetes/kube-addon.md

```diff
@@ -14,6 +14,7 @@ The following topics are discussed:
 * [Network Policy Controller](#npc)
 * [Troubleshooting](#troubleshooting)
  * [Troubleshooting Blocked Connections](#blocked-connections)
+ * [Things to watch out for](#key-points)
 * [Changing Configuration Options](#configuration-options)
 
```

Comment:
change to:  
   * Connection Options to be Aware of

(also list item should be indented as a child of Troubleshooting Blocked Connections)

Reply:
The list is not about connection options; it's all the things we want to tell users about Weave Net on Kubernetes.